tests/integration/auth.spec.coffee

```diff
@@ -3,7 +3,7 @@ m = require('mochainon')
 
 { resin, sdkOpts, credentials, givenLoggedInUser, givenLoggedInUserWithApiKey } = require('./setup')
 
-describe 'SDK authentication', ->
+describe.only 'SDK authentication', ->
```

Comment:
This is good for local testing, but you shouldn't commit it, as it'll disable all the other tests